### DISCUSSION TITLE
Batch Translation

### DISCUSSION THREAD

**Discussion by andwan0** [SubtitleEdit/subtitleedit#6103] [Ideas]
Is there a way to batch translate subtitles? If not, can we get a feature functionality please?

**Comment by mpek** [comment]
Same question here. Seems there isn't. Would be nice and helpful particularly if you have many short videos. 
@andwan0 
Did you open a feature request already?

**Reply by andwan0** [reply]
sorry no

**Comment by Name2889** [comment]
Hi. 

Is it possible to batch translate many files? Could you please help with that?
It would be very convenient, e.g., when translating subtitles of series, etc.

Thank you.
Best regards,
